compiler-rt/test/tysan/lit.cfg.py

```diff
@@ -0,0 +1,161 @@
+# -*- Python -*-
+
+import os
+import platform
+import re
+
+import lit.formats
+
+# Get shlex.quote if available (added in 3.3), and fall back to pipes.quote if
+# it's not available.
+try:
+    import shlex
```

Comment:
`from shlex import quote as sh_quote`?

Reply:
Removed the conditional import, as we currently require Python >= 3.8 https://llvm.org/docs/GettingStarted.html#software